### PR TITLE
Add DLC2 data

### PR DESCRIPTION
**Pull request by KrisXV**
ty to @im-tofa @Karthik99999 @DaWoblefet @smudgerox for helping program, and @Marty-D DarkFE (dont know their github) @Lusamine @kwsch @sora10pls and @sciresm for datamines/research/other behind the scenes work!!!

### CODE REVIEW

File: data/moves.ts [hunk 1]
```diff
@@ -19683,7 +19952,34 @@ export const Moves: {[moveid: string]: MoveData} = {
...
+		num: 906,
+		accuracy: 100,
+		basePower: 120,
+		category: "Special",
+		name: "Tera Starstorm",
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, mirror: 1, noassist: 1, failcopycat: 1, failmimic: 1},
+		onModifyType(move, pokemon) {
+			if (pokemon.species.id === 'Terapagos-Stellar') {
```
Comment (Marty-D) [inline data/moves.ts:19973]:
```suggestion
			if (pokemon.species.name === 'Terapagos-Stellar') {
```

File: data/moves.ts [hunk 2]
```diff
@@ -19683,7 +19952,34 @@ export const Moves: {[moveid: string]: MoveData} = {
...
+				move.type = 'Stellar';
+			}
 		},
+		onModifyMove(move, pokemon) {
+			if (pokemon.species.id === 'Terapagos-Stellar') {
```
Comment (Marty-D) [inline data/moves.ts:19978]:
```suggestion
			if (pokemon.species.name === 'Terapagos-Stellar') {
```

File: data/items.ts [hunk 3]
```diff
@@ -3603,6 +3596,16 @@ export const Items: {[itemid: string]: ItemData} = {
...
 		isNonstandard: "Past",
 	},
+	// TODO - Sprite & Fling Base Power
+	metalalloy: {
+		name: "Metal Alloy",
+		spritenum: 0,
+		fling: {
+			basePower: 30,
+		},
+		num: 2409,
```
Comment (Marty-D) [inline data/items.ts:3606]:
```suggestion
		num: 2482,
```

File: data/moves.ts [hunk 4]
```diff
@@ -11134,6 +11306,23 @@ export const Moves: {[moveid: string]: MoveData} = {
...
+		accuracy: 100,
+		basePower: 100,
+		category: "Special",
+		isNonstandard: "Unobtainable",
+		name: "Malignant Chain",
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		secondary: {
+			chance: 30,
```
Comment (Marty-D) [inline data/moves.ts:11320]:
```suggestion
			chance: 50,
```

File: data/pokedex.ts [hunk 5]
```diff
@@ -18304,6 +18306,145 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
 		battleOnly: "Ogerpon-Cornerstone",
 		forceTeraType: "Rock",
 	},
+	archaludon: {
+		num: 1018,
+		name: "Archaludon",
+		types: ["Steel", "Dragon"],
+		gender: "N",
```
Comment (Marty-D) [inline data/pokedex.ts:18313]:
```suggestion
```

File: data/pokedex.ts [hunk 6]
```diff
@@ -18304,6 +18306,145 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
...
+		prevo: "Duraludon",
+		evoType: "useItem",
+		evoItem: "Metal Alloy",
+		eggGroups: ["Mineral", "Dragon"],
+	},
+	hydrapple: {
+		num: 1019,
+		name: "Hydrapple",
+		types: ["Grass", "Dragon"],
+		gender: "N",
```
Comment (Marty-D) [inline data/pokedex.ts:18328]:
```suggestion
```

File: data/pokedex.ts [hunk 7]
```diff
@@ -18304,6 +18306,145 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
...
+		baseStats: {hp: 106, atk: 80, def: 110, spa: 120, spd: 80, spe: 44},
+		abilities: {0: "Supersweet Syrup", 1: "Regenerator", H: "Sticky Hold"},
+		heightm: 1.8,
+		weightkg: 93,
+		color: "Green",
+		prevo: "Dipplin",
+		evoType: "useItem",
+		evoItem: "Tart Apple",
```
Comment (Marty-D) [inline data/pokedex.ts:18334]:
```suggestion
		evoType: "levelMove",
		evoMove: "Dragon Cheer",
```

File: data/mods/mixandmega/scripts.ts [hunk 8]
```diff
@@ -464,9 +467,8 @@ export const Scripts: ModdedBattleScriptsData = {
 					}
 				}
 			}
-			if (pokemon.species.baseSpecies === 'Ogerpon') {
```
[the remark this answers is not in the document]
Comment (KrisXV) [reply data/mods/mixandmega/scripts.ts:467]: it is

File: config/formats.ts [hunk 9]
```diff
@@ -356,6 +356,15 @@ export const Formats: FormatList = [
 		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Min Source Gen = 9', 'VGC Timer', 'Open Team Sheets'],
 		banlist: [],
 	},
+	{
+		name: "[Gen 9] VGC 2024 Reg F (Bo3)",
+
+		mod: 'gen9',
+		gameType: 'doubles',
+		challengeShow: false,
+		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Min Source Gen = 9', 'VGC Timer', 'Open Team Sheets'],
```
Comment (DaWoblefet) [inline config/formats.ts:365]: `ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Min Source Gen = 9', 'VGC Timer', 'Force Open Team Sheets', 'Best of = 3'],`

**Pull request merged**